### DISCUSSION TITLE
20240104 Weekly Development Meeting

### DISCUSSION THREAD

**Discussion by josecelano** [nautilus-cyberneering/.github#76] [Meetings]
Time: 8:30 GMT+0  Canary Islands Time (9:30 in Vaduz)

Location:
<https://meeting.hodl10.io/ConsistentClassroomsMergeSideways>

Expected duration: 2 to 4 hours with breaks

Communicated attendants:

- CGarnham(@da2ce7)  - x
- JCelano(@josecelano) - x
- CBosse(@cgbosse) - x
- MMelchor(@mario-nt) - x
- GByrne(@grmbyrn) - x
- MVanDijke(@WarmBeer) - x

-------------------------------------------------------------

## Agenda:

1. Everyone will take their turn and:
- Go over his notes added as comments for the meeting which will have been added in advance to give an update of all the work done during the previous week indicating.
	- The issues worked on
	- The progress made on them
	- Other things done
- Inform of any faced challenges and request help if necessary to work on specific issues.
- Highlight the encountered difficulties, especially if there is a risk of them causing delays or affecting end product's quality.
- Communicate what will be the next week's tasks.
- List any new topics for further discussion in an additional Technical meeting that he noted for future discussion in comments to this discussion either before the meeting or after his turn so that they get considered.

During the meeting every team member will take their turns and follow this sequence.
The topics for further discussion will be noted in advance by each the team member and to the meetings agenda.

2. At the end of the meeting if there any new technical topics to be discussed further it will be decided to
set another meeting date for the discussion of these or to hold the meeting hereafter.

Also the persons to be part of the meeting will be decided on so that the others will be free to continue their own work.

**Comment by josecelano** [comment]
### Torrust Tracker

Issues:

- https://github.com/torrust/torrust-tracker/issues/262
- https://github.com/torrust/torrust-tracker/issues/343
- https://github.com/torrust/torrust-tracker/issues/536

PR review:

- https://github.com/torrust/torrust-tracker/pull/561

### Torrust Index

Issues:

- https://github.com/torrust/torrust-index/issues/420. [PR](https://github.com/torrust/torrust-index/pull/425) Pending to merge.

### Torrust Index GUI

PR review:

- https://github.com/torrust/torrust-index-gui/pull/416

### Torrust Website

PR review:

- https://github.com/torrust/torrust-website/pull/127

### Next

Continue fixing bugs and small issues.

- https://github.com/torrust/torrust-index-gui/issues/417
- https://github.com/torrust/torrust-compose/issues/6

**Comment by grmbyrn** [comment]
### Torrust Website
* Share buttons
* Table of contents on wider screens
* Link to latest article in hero of home page
* Copy code button
* Considering which changes are/aren't necessary

### Revision
* Making [BrainScape](https://www.brainscape.com/p/59GDY-LH-CYDRB) flashcards for Git, Svelte, and Python.

**Comment by mario-nt** [comment]
### GUI

- https://github.com/torrust/torrust-index-gui/issues/421
- https://github.com/torrust/torrust-index-gui/pull/423
- https://github.com/torrust/torrust-index-gui/issues/393
- Opened new issues

### Index

- One to one with Jose on https://github.com/torrust/torrust-index/issues/39
- https://github.com/torrust/torrust-index/issues/39

### Training/research
- Tailwind CSS, refs, localStorage, light/dark mode switching and persistance, daisyUI


### Other

- Fixed corrupted virtual machine
- Erasing data and packaging laptop for pickup
